packages/styles/package.json

```diff
@@ -30,24 +30,15 @@
     "format": "prettier src/**/*.scss --write"
   },
   "peerDependencies": {
-    "@angular/core": "^19.0.0",
-    "@ng-bootstrap/ng-bootstrap": "^18.0.0",
-    "bootstrap": "~5.3.0"
+    "@angular/core": "^19.0.0"
```

Comment:
The peer dependency to Angular can be removed as well, it was only there because of ng-bootstrap